components/StyledButton.js

```diff
@@ -1,30 +1,22 @@
-import styled, { css } from "styled-components";
+import styled from "styled-components";
 
 export const StyledButton = styled.button`
   width: 8rem;
   height: 8rem;
   border-radius: 50%;
   background-color: #546391;
   padding: 0.8rem;
- 
   color: #EBE1CA;
   text-decoration: none;
   font-weight: bold;
   border: 0px;
```

Comment:
no need for a unit here :) 

```suggestion
  border: 0;
```